tests_integration/__tests__/debug-print-doc.js

```diff
@@ -3,10 +3,12 @@
 const runPrettier = require("../runPrettier");
 
 describe("prints doc with --debug-print-doc", () => {
-  runPrettier("cli/with-shebang", ["--debug-print-doc"], {
-    input: "0"
-  }).test({
-    stdout: '["0", ";", hardline, breakParent];\n',
+  runPrettier(
+    "cli/with-shebang",
+    ["--parser", "babylon", "--debug-print-doc"],
+    { input: "0" }
+  ).test({
+    stdout: '["0", ";", hardline, breakParent]',
```

Comment:
I'm not sure why this changed, but the new output looks better than the old output, to me

Reply:
I think maybe this printed content was being formatted with the babylon parser before (it was interpreting it as an `ExpressionStatement` > `ArrayExpression` and adding a semicolon)